lib/bom.js

```diff
@@ -261,6 +261,61 @@ declare class Worker extends EventTarget {
     terminate(): void;
 }
 
+declare class SharedWorker extends EventTarget {
+    constructor(stringUrl: string): void;
+    port: MessagePort;
+    onerror: (ev: Event) => any;
+}
+
+declare function importScripts(...urls: Array<string>): void;
+
+declare class WorkerGlobalScope extends EventTarget {
+    self: WorkerGlobalScope;
+    location: WorkerLocation;
+    navigator: WorkerNavigator;
+    importScripts(...urls: Array<string>): void;
+    onerror: (ev: Event) => any;
+    onlanguagechange: (ev: Event) => any;
+    onoffline: (ev: Event) => any;
+    ononline: (ev: Event) => any;
+    onrejectionhandled: (ev: PromiseRejectionEvent) => any;
+    onunhandledrejection: (ev: PromiseRejectionEvent) => any;
+}
+
+declare class DedicatedWorkerGlobalScope extends WorkerGlobalScope {
+    close(): void;
```

Comment:
Ha, interesting! Wonder why it's that way in the spec... it's not a big deal, but I'd say it's probably worth changing if you're in there doing other stuff.